src/screens/Home/Home.tsx

```diff
@@ -38,6 +39,50 @@ interface HomeProps {
 function Home(props: HomeProps) {
   const { classes, lastUpdated, isOnline, currentSite } = props;
 
+  const calculateCustomerData = () => {
+    // customers to charge:  haven't charged yet/ no meterreading
+    // outstanding payments: done the meter reading / charged, haven't paid yet
+    let toCharge = 0;
+    let outstandingPayments = 0;
+    let customers = currentSite.customers;
+    for (let i = 0; i < customers.length; i++) {
+      const currCustomer = customers[i]
+      //depends if the meter readings list is sorted with earliest => latest
+      let latestMeterReading = currCustomer.meterReadings[currCustomer.meterReadings.length - 1];
```

Comment:
Don't think we can depend on the meterReadings list being sorted from earliest to latest (that's not an invariant established yet in our model, but we can). We either need to enforce that "sorted" invariant from the backend or go through all the meter readings and make sure we grab the latest one here. 

Also `const` for `latestMeterReading` since it doesn't change.